.buildkite/pipeline.yml

```diff
@@ -322,71 +322,6 @@
     - docker run --rm --privileged -v /ray/containers:/var/lib/containers -v /ray:/ray --entrypoint /bin/bash
       rayproject/ray-worker-container:nightly-py36-cpu /ray/ci/travis/test-worker-in-container.sh
 
-- label: ":redis: HA GCS (Dashboard + Serve Tests)"
```

Comment:
Maybe in a separate PR, can we leave some tests running without GCS-Ray? At least until we are ready to delete the deprecated codepath.

Reply:
Sounds good.